test/demos/cube-demo-3d/cube/cube-layer.ts

```diff
@@ -186,7 +186,7 @@ export class CubeLayer<
           update: o => o.scale
         }),
         createAttribute({
-          easing: AutoEasingMethod.slerpQuatInOutCubic(1000),
```

Comment:
I'd like the demo to be put back to what it used to be to ensure the scenario I reported is solved under the conditions I found the problem.